README.md

```diff
@@ -1,13 +1,19 @@
-#Docker image to provision a cluster
+#Docker image to provision a docker image registry
 
 ###
 
 ```
-docker build -t setup .
-docker run --env "TOKEN_URL=https://discovery.etcd.io/xxxxxx" --env "VAULT_PASS=xxxxxxxx" --env "DEPLOYER_SERVICE_FILE_LOCATION=https://raw.githubusercontent.com/Financial-Times/coco-docker-image-cache-services/master/service-files/deployer.service" setup
-```
-or if you want to override the AWS credentials
-```
-docker run --env "TOKEN_URL=https://discovery.etcd.io/xxxxxx" --env "VAULT_PASS=xxxxxxxx" --env "DEPLOYER_SERVICE_FILE_LOCATION=https://raw.githubusercontent.com/Financial-Times/coco-docker-image-cache-services/master/service-files/deployer.service" --env "AWS_SECRET_ACCESS_KEY=xxxxxxx" --env "AWS_ACCESS_KEY_ID=xxxxxxxx" setup
+# Build the image
+docker build -t coco/coreos-registry-setup .
```

Comment:
It's fine, just let's stick with one.
https://github.com/Financial-Times/coco-docker-image-cache-provisioner/pull/4/files#diff-04c6e90faac2675aa89e2176d2eec7d8R7